santa/src/before_level.rs

```diff
@@ -93,7 +93,7 @@ impl Picker {
                         .draw(ctx),
                     ]),
                     Widget::row(vec![
-                        Widget::draw_svg(ctx, "system/assets/tools/mouse.svg"),
+                        Icon::from_path("system/assets/tools/mouse.svg").into_widget(ctx),
```

Comment:
Looks like an older version of your refactor -- these should be `Image::icon(...).into_widget`

Reply:
ah geeze, sorry santa. Thanks for catching. I've rebased and applied a fixup commit to address this.

Reply:
Santa forgives you.